doc-pages/webpack/config.js

```diff
@@ -1,9 +1,6 @@
-const {resolve, join} = require('path');
+const {resolve} = require('path');
 const webpack = require('webpack');
-
-const rootDir = join(__dirname, '../..');
-const demoDir = join(__dirname, '..');
-const demoSources = join(demoDir, 'src');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
 
 // Otherwise modules imported from outside this directory does not compile
```

Comment:
No idea. This is a carry-over from the deck.gl stuff. It was pulling in files from the main `/src` to use as examples. If we agree to install from `npm` instead (like a regular app) then this won't be a problem. We can tweak this as the docs become more finished.